org.munadi.Munadi.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "munadi",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config/autostart:create",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "munadi",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 org.munadi.Munadi.desktop /app/share/applications/org.munadi.Munadi.desktop",
+                "install -Dm644 org.munadi.Munadi.appdata.xml /app/share/appdata/org.munadi.Munadi.appdata.xml",
+                "tar xf org.munadi.Munadi.icons.tar.gz -C /app/share/",
+                "mv src/* .",
```

Comment:
???

Reply:
I couldn't cd into src then invoke qmake, source code is under src dir, how can I do it?
I could do qmake src/munadi.pro if that's better?

Reply:
Found a better way [1], will update.
1. http://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest

Reply:
Done